contracts/LemonadeUniqueCollectionV1.sol

```diff
@@ -16,10 +16,25 @@ contract LemonadeUniqueCollectionV1 is IMintable {
         string memory description,
         string memory tokenPrefix
     ) payable {
-        collection = ICollectionHelpers(collectionHelpers)
-            .createNonfungibleCollection(name, description, tokenPrefix);
+        ICollectionHelpers collectionHelpers_ = ICollectionHelpers(
+            collectionHelpers
+        );
+
+        collection = collectionHelpers_.createNFTCollection{value: msg.value}(
+            name,
+            description,
+            tokenPrefix
+        );
+        collectionHelpers_.makeCollectionERC721MetadataCompatible(
+            collection,
+            ""
+        );
+
+        ICollection collection_ = ICollection(collection);
 
-        ICollection(collection).setTokenPropertyPermission(
+        collection_.addCollectionAdmin(address(this));
```

Comment:
I guess it would be good to add here also
`collection_.addToCollectionAllowList(address(this));`

To make it possible to mint tokens to the collection from the contract